### PR TITLE
Add support for manga activity texts

### LINKED ISSUE #127: Activities without text
**Describe the bug**
Some activities, namely activities that involve non-anime media, have no text thus showing just the image and empty space.

**To Reproduce**
1. Update something on your list involving Manga
2. Go to ZeroTwo Home
3. See your latest activities
4. The latest manga activity has no text

**Expected behavior**
Activity should have text

**Screenshots**
image

### PR DESCRIPTION
**Pull request by NicoAiko**
Fixes #127

### CODE REVIEW

File: src/assets/locales/pt_BR.json [hunk 1]
```diff
@@ -74,13 +74,21 @@
...
           "plansToWatch": "Planeja assistir {0}",
           "watchedEpisode": "Assistido episódio {1} de {0}",
           "dropped": "Dropou {0}",
           "rewatchedEpisode": "Reassistindo episódio {1} of {0}",
-          "pausedWatching": "Pausou {0}"
+          "pausedWatching": "Pausou {0}",
+          "readChapter": "Read chapter {1} of {0}",
+          "plansToRead": "Plans to read {0}",
+          "rereadChapter": "Reread chapter {1} of {0}",
+          "pausedReading": "Paused reading {0}"
```
Comment (frozaum) [inline src/assets/locales/pt_BR.json]: Pausada a leitura {0}

File: src/assets/locales/zh_CN.json [hunk 2]
```diff
@@ -74,13 +74,21 @@
     "aniList": {
       "home": {
         "activities": {
+          "badges": {
+            "anime": "Anime",
```
Comment (Mornwind) [inline src/assets/locales/zh_CN.json]: "anime": "动画",

File: src/assets/locales/zh_CN.json [hunk 3]
```diff
@@ -74,13 +74,21 @@
...
+            "manga": "Manga"
```
Comment (Mornwind) [inline src/assets/locales/zh_CN.json]: 漫画/小说

File: src/assets/locales/zh_CN.json [hunk 4]
```diff
@@ -74,13 +74,21 @@
...
+          },
           "headline": "最新动态",
           "completed": "已看完 {0}",
           "plansToWatch": "准备看 {0}",
           "watchedEpisode": "已看 {0} 的第 {1} 话",
           "dropped": "放弃看 {0}",
           "rewatchedEpisode": "重温 {0} 的第 {1} 话",
-          "pausedWatching": "暂时不看 {0}"
+          "pausedWatching": "暂时不看 {0}",
+          "readChapter": "Read chapter {1} of {0}",
```
Comment (Mornwind) [inline src/assets/locales/zh_CN.json]: 已看 {0} 的第 {1} 章节

File: src/assets/locales/zh_CN.json [hunk 5]
```diff
@@ -74,13 +74,21 @@
...
+          "plansToRead": "Plans to read {0}",
```
Comment (Mornwind) [inline src/assets/locales/zh_CN.json]: 准备看 {0}

File: src/assets/locales/zh_CN.json [hunk 6]
```diff
@@ -74,13 +74,21 @@
...
+          "rereadChapter": "Reread chapter {1} of {0}",
```
Comment (Mornwind) [inline src/assets/locales/zh_CN.json]:
```suggestion
          "rereadChapter": "重温 {0} 的第 {1} 章",
```

File: src/assets/locales/zh_CN.json [hunk 7]
```diff
@@ -74,13 +74,21 @@
...
+          "pausedReading": "Paused reading {0}"
```
Comment (Mornwind) [inline src/assets/locales/zh_CN.json]:
```suggestion
          "pausedReading": "暂时不看 {0}"
```

File: src/assets/locales/ar.json [hunk 8]
```diff
@@ -74,13 +74,21 @@
     "aniList": {
       "home": {
         "activities": {
+          "badges": {
+            "anime": "Anime",
+            "manga": "Manga"
```
Comment (nattadasu) [inline src/assets/locales/ar.json]:
```suggestion
            "manga": "مانغا"
```

File: src/assets/locales/ar.json [hunk 9]
```diff
@@ -74,13 +74,21 @@
     "aniList": {
       "home": {
         "activities": {
+          "badges": {
+            "anime": "Anime",
```
Comment (nattadasu) [inline src/assets/locales/ar.json]:
```suggestion
            "anime": "أنمي",
```

File: src/assets/locales/ar.json [hunk 10]
```diff
@@ -74,13 +74,21 @@
...
+          },
           "headline": "أحدث الأنشطة",
           "completed": "منجز {0}",
           "plansToWatch": "مخطط للمشاهدة {0}",
           "watchedEpisode": "شاهد الحلقة {1} من {0}",
           "dropped": "توقفت {0}",
           "rewatchedEpisode": "حلق المشاهدة الحلقة {1} من {0}",
-          "pausedWatching": "توقف {0}"
+          "pausedWatching": "توقف {0}",
+          "readChapter": "Read chapter {1} of {0}",
```
Comment (nattadasu) [inline src/assets/locales/ar.json]:
```suggestion
          "readChapter": "قرأت الفصل {1} من {0}",
```

File: src/assets/locales/ar.json [hunk 11]
```diff
@@ -74,13 +74,21 @@
...
+          "plansToRead": "Plans to read {0}",
```
Comment (nattadasu) [inline src/assets/locales/ar.json]:
```suggestion
          "plansToRead": "خطط لقرء {0}",
```

File: src/assets/locales/ar.json [hunk 12]
```diff
@@ -74,13 +74,21 @@
...
+          "rereadChapter": "Reread chapter {1} of {0}",
```
Comment (nattadasu) [inline src/assets/locales/ar.json]:
```suggestion
          "rereadChapter": "حلق القراءة الفصل {1} من {0}",
```

File: src/assets/locales/ar.json [hunk 13]
```diff
@@ -74,13 +74,21 @@
...
+            "manga": "Manga"
+          },
           "headline": "أحدث الأنشطة",
           "completed": "منجز {0}",
           "plansToWatch": "مخطط للمشاهدة {0}",
           "watchedEpisode": "شاهد الحلقة {1} من {0}",
           "dropped": "توقفت {0}",
           "rewatchedEpisode": "حلق المشاهدة الحلقة {1} من {0}",
-          "pausedWatching": "توقف {0}"
+          "pausedWatching": "توقف {0}",
```
Comment (nattadasu) [inline src/assets/locales/ar.json]:
```suggestion
          "pausedWatching": "توقف المشاهدة {0}",
```

File: src/assets/locales/ar.json [hunk 14]
```diff
@@ -74,13 +74,21 @@
...
+          "readChapter": "Read chapter {1} of {0}",
+          "plansToRead": "Plans to read {0}",
+          "rereadChapter": "Reread chapter {1} of {0}",
+          "pausedReading": "Paused reading {0}"
```
Comment (nattadasu) [inline src/assets/locales/ar.json]:
```suggestion
          "pausedReading": "توقف القراءة {1}"
```

File: src/assets/locales/id.json [hunk 15]
```diff
@@ -74,13 +74,21 @@
...
+            "manga": "Manga"
+          },
           "headline": "Aktivitas Terbaru",
           "completed": "Selesai {0}",
           "plansToWatch": "Direncanakan {0}",
           "watchedEpisode": "Episode {1} dari {0} selesai",
           "dropped": "Dihentikan {0}",
           "rewatchedEpisode": "Episode {1} dari {0} ditonton kembali",
-          "pausedWatching": "Ditahan {0}"
+          "pausedWatching": "Ditahan {0}",
```
Comment (nattadasu) [inline src/assets/locales/id.json]:
```suggestion
          "pausedWatching": "Menahan menonton {0}",
```

File: src/assets/locales/id.json [hunk 16]
```diff
@@ -74,13 +74,21 @@
...
+          "readChapter": "Read chapter {1} of {0}",
```
Comment (nattadasu) [inline src/assets/locales/id.json]:
```suggestion
          "readChapter": "Membaca chapter {1} dari {0}",
```

File: src/assets/locales/id.json [hunk 17]
```diff
@@ -74,13 +74,21 @@
...
+          "plansToRead": "Plans to read {0}",
```
Comment (nattadasu) [inline src/assets/locales/id.json]:
```suggestion
          "plansToRead": "Direncanakan membaca {0}",
```

File: src/assets/locales/id.json [hunk 18]
```diff
@@ -74,13 +74,21 @@
...
+          "rereadChapter": "Reread chapter {1} of {0}",
```
Comment (nattadasu) [inline src/assets/locales/id.json]:
```suggestion
          "rereadChapter": "Membaca kembali chapter {1} dari {0}",
```

File: src/assets/locales/id.json [hunk 19]
```diff
@@ -74,13 +74,21 @@
...
+          "pausedReading": "Paused reading {0}"
```
Comment (nattadasu) [inline src/assets/locales/id.json]:
```suggestion
          "pausedReading": "Menahan membaca {0}"
```

File: src/assets/locales/ms.json [hunk 20]
```diff
@@ -74,13 +74,21 @@
...
+          },
           "headline": "Aktiviti Terkini",
           "completed": "Selesai {0}",
           "plansToWatch": "Dirancang {0}",
           "watchedEpisode": "Episod {1} dari {0} selesai",
           "dropped": "Berhenti {0}",
           "rewatchedEpisode": "Episod {1} dari {0} ditonton kembali",
-          "pausedWatching": "Ditahan {0}"
+          "pausedWatching": "Ditahan {0}",
+          "readChapter": "Read chapter {1} of {0}",
```
Comment (nattadasu) [inline src/assets/locales/ms.json]:
```suggestion
          "readChapter": "Baca bab {1} daripada {0}",
```

File: src/assets/locales/ms.json [hunk 21]
```diff
@@ -74,13 +74,21 @@
...
+          "plansToRead": "Plans to read {0}",
```
Comment (nattadasu) [inline src/assets/locales/ms.json]:
```suggestion
          "plansToRead": "Rancang membaca {0}",
```

File: src/assets/locales/ms.json [hunk 22]
```diff
@@ -74,13 +74,21 @@
...
+          "rereadChapter": "Reread chapter {1} of {0}",
```
Comment (nattadasu) [inline src/assets/locales/ms.json]:
```suggestion
          "rereadChapter": "Chapter {1} daripada {0} dibaca kembali",
```

File: src/assets/locales/ms.json [hunk 23]
```diff
@@ -74,13 +74,21 @@
...
+          "pausedReading": "Paused reading {0}"
```
Comment (nattadasu) [inline src/assets/locales/ms.json]:
```suggestion
          "pausedReading": "Menjeda membaca {0}"
```

File: src/assets/locales/ms.json [hunk 24]
```diff
@@ -74,13 +74,21 @@
...
+            "manga": "Manga"
+          },
           "headline": "Aktiviti Terkini",
           "completed": "Selesai {0}",
           "plansToWatch": "Dirancang {0}",
           "watchedEpisode": "Episod {1} dari {0} selesai",
           "dropped": "Berhenti {0}",
           "rewatchedEpisode": "Episod {1} dari {0} ditonton kembali",
-          "pausedWatching": "Ditahan {0}"
+          "pausedWatching": "Ditahan {0}",
```
Comment (nattadasu) [inline src/assets/locales/ms.json]:
```suggestion
          "pausedWatching": "Menjeda menonton {0}",
```

**Pull request merged**